### DISCUSSION TITLE
[Feature] Please upload snap to Snap Store

### DISCUSSION THREAD

**Discussion by MCPE-PC** [bridge-core/bridge.#172] [Ideas]
**Summary**
To install the snap file, I should use `--dangerous` option
```bash
mcpepc@mcpe-pc-laptop:~$ snap install bridge.snap
error: cannot find signatures with metadata for snap "bridge.snap"
mcpepc@mcpe-pc-laptop:~$ snap install --dangerous bridge.snap
bridge:icon-themes 을(를) gtk-common-themes:icon-themes에 연결하십시오          bridge 1.4.3 is installed.
```
And I cannot refresh the snap automatically without assert file

**Alternatives/Considerations**
I can also just use AppImage

**Comment by montudor** [comment]
@MCPE-PC sorry for the delay. We're looking into this. I'll be sure to provide updates here once we have more information.
